src/app/helpers/pgp.js

```diff
@@ -42,7 +42,7 @@ export const allKeysExpired = async (keys = []) => {
     const keyObjects = keys.map((publicKey) => isExpiredKey(publicKey));
     const isExpired = await Promise.all(keyObjects);
 
-    return isExpired.some((keyExpired) => !keyExpired);
+    return !isExpired.some((keyExpired) => !keyExpired);
```

Comment:
This line can be improve, I'm seeing a double negation here.

Reply:
We can write
```
return isExpired.every((keyExpired) => keyExpired)
```
Although that's more clear syntax, I would say it's a better practice to use `!some`, because `every` has to check all the array entries, while `some` will return `true` as soon as one entry is `true`.

In this case it does not make a lot of a difference because `isExpired` should be a small array. I can change it if you think that's better